package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "handset-detect",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

Comment:
need to upgrade the major version, as this has a breaking change to anyone using the API (changed hosted to cache in options)